src/app/containers/PageHandlers/withData.jsx

```diff
@@ -1,39 +1,65 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { shape, element } from 'prop-types';
 import articlePropTypes from '../../models/propTypes/article';
 import ErrorMain from '../ErrorMain';
 import nodeLogger from '../../helpers/logger.node';
 import deepGet from '../../helpers/json/deepGet';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import idSanitiser from '../../lib/utilities/idSanitiser';
 
 const logger = nodeLogger(__filename);
 
-// checks for data and status keys, setting default status if not found
+/* used to remove - & _ from locale and service language for comparison
+eg: transforms en-GB, en_gb to engb, etc.
+TODO: move this to a util ? not sure where exactly
+*/
+const sanitizedLocaleToLowerCase = text => idSanitiser(text.toLowerCase());
+
+// checks for data, status & language keys, setting default status if not found
 const constructRenderObject = data => ({
   status: deepGet(['status'], data) || 500,
   pageData: deepGet(['pageData'], data),
+  language: deepGet(['pageData', 'metadata', 'passport', 'language'], data),
```

Comment:
Good work so far with looking at the JSON data and extracting values. 

Instead of checking the language value in the passport, we should check the `home` value instead. This has a string of the format `http://www.bbc.co.uk/ontologies/passport/home/News`. 


The reason we should check the home field is because later on when we expand to multiple other services, there can be multiple services with the same language. For example, News and Sport. These will have the same language, but they will be different services.  

So if you could check the `home` string and compare the sanitised value at the end of that URL - that should be the same as the `service` value. `const { service } = useContext(ServiceContext);`

Reply:
thanks, @sareh that totally makes sense